tsconfig.json

```diff
@@ -8,7 +8,7 @@
     "allowSyntheticDefaultImports": true,
     "esModuleInterop": true,
     "resolveJsonModule": true,
-    "moduleResolution": "node",
```

Comment:
If `moduleResolution` is set to `bundler`, certain dependencies cannot be imported. e.g. `youtubei.js`.

Reply:
can you ellaborate on that?
any other `moduleResolution` introduces issues

Reply:
looks like it wasn't a big deal